astroquery/heasarc/core.py

```diff
@@ -600,8 +792,9 @@ def download_data(self, links, host='heasarc', location='.'):
         ----------
         links : `astropy.table.Table` or `astropy.table.Row`
```

Comment:
presumably you called this parameter `links` because these suppose to be datalinks? If yes, please add one more sentence that explains it.